doc/changelog/11-infrastructure-and-dependencies/13617-make+use_dune_for_ocaml.rst

```diff
@@ -0,0 +1,9 @@
+- **Changed:**
+  Coq's OCaml parts and tools [coq-core] are now built using Dune.
+  The main user-facing change is that Dune >= 2.5 is now required to
+  build Coq. For developers and plugin authors, see the entry in
+  doc/dev/changes.md
+  - the following makefile targets have been removed: byte install-byte
+  - Bugs fixed: # , # , #
```

Comment:
Left-over placeholders.
IMHO, you can just remove the line, this is not important info.

Reply:
Done, improved credits.